packages/next/src/index.ts

```diff
@@ -1072,6 +1072,7 @@ export const build: BuildV2 = async ({
           handler: '___next_launcher.cjs',
           runtime: nodeVersion.runtime,
           ...lambdaOptions,
+          operationType: 'SSR',
```

Comment:
Is this just being added for testing? This specific line is for legacy Next.js apps (`< 7.0.2`)

Reply:
It was added for completeness. If we don't add this and someone deploys an old next app, the richer deployments output will show an "API" label instead.

Reply:
What about existing deployments that are empty? Is there going to be a migration or just leave them empty?